web/src/main/resources/static/index.html

```diff
@@ -12,64 +12,138 @@
+              <br/>
+              <span>Oncotree verison currently displayed: </span>
+              <span id="other-version">
+                <select class="other-version-content"></select>
+              </span>
+                <br/>
+            </div>
+          </div>
           <br/>
-          <span>Oncotree verison currently displayed: </span>
-          <span id="other-version">
-            <select class="other-version-content"></select>
+          <div class="row">
+            <div class="col-lg-4 col-md-5 col-sm-6 col-xs-9 has-addon-feedback">
+              <div class="input-group" id='tumor_search'>
+                <input type="text" class="form-control" placeholder="Search Tumor Type" />
+                <span id="searchRemoveIcon" class="form-control-feedback glyphicon glyphicon-remove-circle"></span>
+                <span id="searchResult" class="form-control-feedback result"></span>
+                <span class="input-group-btn">
+                  <button type="button" class="btn btn-default">Search</button>
+                </span>
+
+              </div>
+            </div>
+            <div class="col-lg-3 col-md-3 col-sm-3 col-xs-3">
+              <div class="btn-group">
+                <button type="button" class="btn btn-default" id='expand-nodes-btn'>
+                  <span class="glyphicon glyphicon-resize-full"></span>
+                </button>
+
+                <button type="button" class="btn btn-default" id='collapse-nodes-btn'>
+                  <span class="glyphicon glyphicon-resize-small"></span>
+                </button>
+              </div>
+            </div>
+          </div>
+          <div id="tree"></div>
+          <div id="summary-duplicates" class="col-lg-6 col-md-8 col-sm-12 col-xs-12">
+            <b>Acronym crash:</b>
+            <p id="" class="bg-danger"></p>
+          </div>
+          <span id="top-link-block" class="hidden">
+            <a href="#top" class="well well-sm"  onclick="$('html,body').animate({scrollTop:0, scrollLeft: 0},'slow');return false;">
+                <i class="glyphicon glyphicon-chevron-up"></i> Back to Top
+            </a>
           </span>
-            <br/>
         </div>
-      </div>
-      <br/>
-      <div class="row">
-        <div class="col-lg-4 col-md-5 col-sm-6 col-xs-9 has-addon-feedback">
-          <div class="input-group" id='tumor_search'>
-            <input type="text" class="form-control" placeholder="Search Tumor Type" />
-            <span id="searchRemoveIcon" class="form-control-feedback glyphicon glyphicon-remove-circle"></span>
-            <span id="searchResult" class="form-control-feedback result"></span>
-            <span class="input-group-btn">
-              <button type="button" class="btn btn-default">Search</button>
-            </span>
 
+        <!-- news tab -->
+        <div class="tab-pane fade" id='news' role="tabpanel">
+          <div class="embed-responsive embed-responsive-16by9">
```

Comment:
This div and iframe aren't needed since news is loaded dynamically and put inside the id='news' div.